resources/docker-build_p2p/ssh.rc

```diff
@@ -9,7 +9,7 @@ http_port=800
 
 #### Configuration received from th server
 ### the port of the sshd service in the p2p server
-p2p_port=
+p2p_port=2201
```

Comment:
Weird, but from another point of view: Wouldn't it be more consistent if any other "variable" has an initial value, that this one also has one? :)